src/GraphQL/DataObjectInputProcessor/ManyToManyObjectRelation.php

```diff
@@ -42,8 +42,13 @@ public function process(Concrete $object, $newValue, $args, $context, ResolveInf
                     if (isset($newValueItemValue["type"]) && $newValueItemValue["type"] !== 'object') {
                         throw new ClientSafeException("expected object type");
                     }
-
-                    $element = \Pimcore\Model\Element\Service::getElementById('object', $newValueItemValue["id"]);
+                    
+                    if (isset($newValueItemValue["id"])) {
```

Comment:
maybe a candidate for a trait

Reply:
I added the trait 'ElementIdentificationTrait' which gets the element from Service::getElementById() or Service::getElementByPath()). Both functions do handle objects, assets and documents.